app/javascripts/helpers/web3Service.jsx

```diff
@@ -256,7 +256,13 @@ class Web3Service {
           reject(err);
         } else {
           const paddingIndex = slotName.substring(1).indexOf('0');
-          const partSlotName = slotName.substring(0, paddingIndex + 1);
+          let partSlotName;
+          if (paddingIndex === -1) {
```

Comment:
Because of encoding from unicode to hexcode.
At contract, slotName will be saved at 16 bytes.
when slotName's data is under 16 bytes, it will be filled with '0' hex code.
But, we found there is some unicode that has '0' hex code when it encoded.
So, that code will be changed soon :D